src/view-impl-common.c

```diff
@@ -52,6 +55,21 @@ view_impl_unmap(struct view *view)
 	if (view == server->active_view || !server->active_view) {
 		desktop_focus_topmost_view(server);
 	}
+
+	desktop_update_top_layer_visibility(view->server);
```

Comment:
Hm.. stumbled over the removal of the fullscreen check here but I guess it actually makes sense to not check for it: the window might have been the only one in front of a fullscreen window.